index.html

```diff
@@ -38,7 +38,7 @@ <h3>I'm a Software Developer.</h3>
                 <a href=""> <img src="images/linkedin.svg" alt="linkedin"></a>
                 <a href=""> <img src="images/peace.svg" alt="peace"></a>
                 <a href="">  <img  src="images/twitter.svg" alt="twitter"></a>
-                <a href=""> <img src="images/Icon - Medium.svg" alt="medium"></a>
+                <a href=""> <img src="images/Icon-Medium.svg" alt="medium"></a>
```

Comment:
Kindly wrap the social media section in a `ul` and make each element inside an `li` element , this will make your code look more semantic 👍 😄 😄